index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav class="nav-bar">
+      <a href="/" class="logo">My Logo</a>
+      <img class="menu" src="/img/Union.svg " alt="" />
+    </nav>
```

Comment:
- [ ] Great job 👍, It would be better to wrap the `nav` in a `header` tag to improve the structure and semantics.